### ISSUE TITLE
Add ability to append custom message to registration response

### ISSUE THREAD

**Issue by woogerboy21** [Cockatrice/Cockatrice#1863]
It would be nice to have a server ini option that would be appended to the dialogue that is presented when a user registered to a server .Might take a bit of thinking on how to do this.  Maybe in the response PB?

**Comment by Daenyth** [comment]
We'll need a new response, I think.

This is reasonable.

We should add whitelists and blacklists.

**Comment by tooomm** [comment]
can this be closed after #2398 @woogerboy21 ?

**Comment by woogerboy21** [comment]
@tooomm Nope not yet, #2398 only added the protocol portion needed to make this happen.  Now I can apply the actual server side code logic to add the functionality :)
